### DISCUSSION TITLE
Implement a middleware on specifi column

### DISCUSSION THREAD

**Discussion by saeed-rostami** [nuwave/lighthouse#2572] [General]
Thanks for this great package...is it possible to implement a middleware on a specific column for a type schema?
for example, I have Episode type...I want only auth users access to the file column...otherwise, the file returns false...
something like below...
thank you a lot

```

type Episode @model(class: "Modules\\PodcastApp\\Models\\Episode") {
    id: ID!
    **file: EpisodeFile @hasOne @middleware("custom-middleware")**
    title: String!
    description: String
    cover: String!
    season: Season! @belongsTo
    channel : Channel @method(name: "channel")
}
```

**Comment by spawnia** [comment]
https://lighthouse-php.com/master/security/authentication.html#guard-selected-fields

**Reply by saeed-rostami** [reply]
My point is on a specific column on type ...
Not on whole query...thanks

**Reply by spawnia** [reply]
Yup, and that is entirely possible with `@guard`. It can go on any field.
